crates/sync/src/strategy.rs

```
@@ -0,0 +1,21 @@
+    ///
+    /// # Parameters
+    ///
+    /// * `tick` - The current tick count of the locking loop.
+    fn relax(tick: usize);
+}
+
+pub struct Spin;
+
+impl RelaxStrategy for Spin {
+    #[track_caller]
+    fn relax(tick: usize) {
```

Clippy (1.96.0)

[warning] unused variable

(warning)